open-api/rest-catalog-open-api.yaml

```diff
@@ -2508,19 +2517,6 @@ components:
             }
           }
 
-    IcebergErrorResponse:
```

Comment:
Didn't mean to delete it but it was unused can add back in a revision

Reply:
I think the response here is for the `responses` section, versus the other one is for the `schemas` section, so we cannot remove this